ansible/vars/icds/icds_public.yml

```diff
@@ -135,10 +135,12 @@ postgresql_dbs:
   - name: commcarehq_ucr
     host: "{{ groups.postgresql.0 }}"
     django_alias: ucr
+    django_migrate: False
   - django_alias: icds-ucr
     name: commcarehq_icds_aggregatedata
     host: "{{ groups.postgresql.0 }}"
     query_stats: True
+    django_migrate: False
```

Comment:
@snopoke It's possible that we do want to migrate this https://github.com/dimagi/commcare-hq/tree/master/custom/icds_reports/migrations

I think we have moved most of the table generation into sql scripts that run druing the aggregation, but historically there have been migrations on icds-ucr. @sheelio can you confirm

Reply:
abf9e86